### ISSUE TITLE
AuditLog and Security?

### ISSUE THREAD

**Issue by artfuld** [ActiveForums/ActiveForums#256]
Hello!
Going through the tables and it looks like there is some legacy tables that haven't been removed.

Can anyone tell me what the table AuditLog is meant for? I have a very old DNN/AF from 2007 and it doesn't have anything in it. When I search the code I can't find any references to it and there's no Stored procedures for it.

Might be a good idea to remove this unless someone can come up with a reason for it.

**Comment by supermansrf** [comment]
I would love for this to work.  I would like for it to show if someone edits their own post, if a moderator edits someone's post, if a post gets deleted, moved, split etc.  

I'm betting that it was a planned feature that never got implemented but its hard telling.  We were dealing with a problem one day and I ran across this in the database.  I had initially gotten excited thinking that it was keeping track of things but then saw it was blank and disappointed.
